2-With-broker/UserDetailsClient/UserDetailsClient/App.cs

```diff
@@ -29,10 +29,6 @@ public class App : Application
 
         public App()
         {
-            PCA = PublicClientApplicationBuilder.Create(ClientID)
```

Comment:
Why is this removed?

Reply:
This is created only once and this creation is with no broker. And It is created when needed.